index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/css/bootstrap.min.css"
+      integrity="sha384-MCw98/SFnGE8fJT3GXwEOngsV7Zt27NXFoaoApmYm81iuXoPkFOJwJ8ERdknLPMO"
+      crossorigin="anonymous" />
+    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css" />
+    <title>Book Catalog</title>
+  </head>
+
+  <body>
+    <main>
+      <section
+        class="container search-id">
+        <h3>Search Book By Id</h3>
+        <form action="#">
+          <input
+            type="text"
+            class="form-control input-search-id"/>
```

Comment:
similarly, this don't use `-id`, if your purpose is to define a selector for javascript purpose then use `js-<anything>`. I have already explained this in the chat.

Reply:
done